### DISCUSSION TITLE
https://www.xiaogenban1993.com/blog/24.07/%E7%90%86%E8%B4%A2%E6%8A%95%E8%B5%84

### DISCUSSION THREAD

**Discussion by giscus** [sunwu51/notebook#58] [General]
# https://www.xiaogenban1993.com/blog/24.07/%E7%90%86%E8%B4%A2%E6%8A%95%E8%B5%84

Generated by FreeCoder

https://www.xiaogenban1993.com/blog/24.07/%E7%90%86%E8%B4%A2%E6%8A%95%E8%B5%84

**Comment by sunwu51** [comment]
7月16日 出来美联储9月100%降息的消息，黄金一度冲高历史最高，立马卖一半，夜盘黄金期货跌0.4%，预计周四周五都会有回调，伴随的是获益方出货，放量下跌，同时引起恐慌从众卖出，此时可以再次买入，因为9月降息真正落地还会有一波上涨，为这一波上涨建仓。

![img](https://i.imgur.com/dJArw1a.png)

**Reply by cryboy007** [reply]
up这波纳指回调你怎么看，都说英伟达是卖铲子的所以业绩好,最近二周我买的英伟达.但是买铲子的都没挖出金子(谷歌,tesla). 相关的芯片etf已经回调了15%.有点难受收益快亏没了

**Reply by cryboy007** [reply]
up这波纳指回调你怎么看，都说英伟达是卖铲子的所以业绩好,但是买铲子的都没挖出金子(谷歌,tesla).最近二周我买的英伟达 相关的芯片etf已经回调了15%.有点难受收益快亏没了

**Reply by sunwu51** [reply]
我已经亏麻了

**Reply by cryboy007** [reply]
哈哈哈,我也是。上涨的时候唯唯诺诺加仓幅度很小从去年6月份建仓，今年2月份3层仓位,结果在6月份实在没忍住就重仓了.结果就开始下跌趋势了。不过相比较于美股我反而更看好印度（人口）.主要还是美股涨的太多了

**Reply by sunwu51** [reply]
印度股票不是很主流，我纳斯达克定投了快5年了，总体还是上涨的。黄金也定投了蛮久了，坚持坚持。现在把钱放到债券了一部分。

**Comment by pcp0606** [comment]
UP可以建个交流群吗？

**Reply by sunwu51** [reply]
q群 991442426，群里不是很活跃

**Comment by pcp0606** [comment]
UP怎么看待A股这波上涨

**Reply by sunwu51** [reply]
我一点没赚着，气得我直跺脚
